client/post-editor/editor-sidebar/style.scss

```diff
@@ -69,3 +74,63 @@
 		display: none;
 	}
 }
+
+.editor-sidebar__parent-region {
+	display: flex;
+	flex-direction: column;
+	position: absolute;
+	top: 47px;
+	bottom: 47px;
+	width: 100%;
+	transition: all 150ms cubic-bezier(0.075, 0.82, 0.165, 1);
```

Comment:
The idea was to limit the divergence with the transition line 11 (used to show/hide the whole sidebar). In an ideal world I would fix it there as well and refactor the file to not have that dup, but I also want to limit as much as possible janitorial changes ...

I've updated the usage introduced by this PR, and added an item on my todo to potentially fix/refactor this file in a smaller more-janitorial-y-focused PR.